src/files.ts

```diff
@@ -348,11 +345,11 @@ export async function pushFiles(silent = false) {
       if (!silent) spinner.stop(true);
       // no error
       if (!silent) {
-        nonIgnoredFilePaths.forEach((filePath: string) => {
-          console.log(`└─ ${filePath}`);
-        });
+        console.log(listFilesHelper(nonIgnoredFilePaths));
         console.log(LOG.PUSH_SUCCESS(nonIgnoredFilePaths.length));
       }
     }
   });
 }
+
+export const listFilesHelper = (files: string[]) => files.map(file => `└─ ${file}`).join(`\n`);
```

Comment:
- Change `listFilesHelper` to `logFileList`
- `console.log` the fileMapString in this method.